### PR TITLE
Update /node-red hero section

### PR DESCRIPTION
**Pull request by Yndira-FlowForge**
## Description

- [x]  Replace hero content with: Get started online for free + Node-RED hosted image + Sign up button
- [x]  Fold FAQs
- [x]  Add a link to /customer-stories/ (as a first iteration)

## Related Issue(s)

closes [#1315](https://github.com/FlowFuse/website/issues/1315)

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowforge/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/node-red/index.njk [hunk 1]
```diff
@@ -2,79 +2,46 @@
 layout: page
 nohero: true
 meta:
-    title: Node-RED Resources
+    title: Node-RED online for free
```
Comment (ZJvandeWeg) [inline src/node-red/index.njk:5]: It's not for free?

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

File: src/node-red/index.njk [hunk 2]
```diff
@@ -162,12 +129,22 @@ sitemapPriority: 0.8
...
     </div>
+</div>    
 
-    <div class="text-white">
-        {% include "components/dividers/light-grey-curve.svg" %}
-    </div>
+<!--Customer Stories Banner-->
+<a href="/customer-stories/" class="w-full bg-teal-700 text-center text-teal-50 px-6 py-3 text-lg flex flex-row items-center justify-center gap-1 hover:bg-teal-800 hover:no-underline cursor-pointer flex-wrap">
+    Check out some
+    <span class="font-bold hover:no-underline"> Node-RED </span>
```
Comment (ZJvandeWeg) [inline src/node-red/index.njk:137]: Having the spaces inside the span I don't think is needed?

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

File: src/node-red/index.njk (hunk 2 above)
Comment (Yndira-FlowForge) [inline src/node-red/index.njk:137]:
```suggestion
    <span class="font-bold hover:no-underline">Node-RED</span> 
```

File: src/node-red/index.njk (hunk 1 above)
Comment (Yndira-FlowForge) [reply src/node-red/index.njk:5]: How do you think we should phrase it? Just Node-RED online?

Comment (ZJvandeWeg) [reply src/node-red/index.njk:5]: "Cloud hosted Node-RED"? Online I don't believe is nice either -- My laptop is online, but it's not an environment I want to run Node-RED for professional use-cases.

File: src/node-red/index.njk (hunk 2 above)
Comment (Yndira-FlowForge) [reply src/node-red/index.njk]: You're right, the gap-1 does the work, I removed the spaces

File: src/node-red/index.njk (hunk 1 above)
Comment (ZJvandeWeg) [reply src/node-red/index.njk:5]:
```suggestion
    title: Start using Node-RED online for free
```

**Pull request merged**